src/compiler/third_party/tiny-sha1/TinySHA1.hpp

```diff
@@ -0,0 +1,197 @@
+/*
```

Comment:
It is a little bit annoying to have this in addition to src/sha1.{cc,h}.

Reply:
as discussed: for now we will keep it, but we will refactor it to use the src/sha1.{cc.h}.